pilot/pkg/networking/core/v1alpha3/listener_builder.go

```diff
@@ -568,6 +568,7 @@ func buildInboundCatchAllFilterChains(configgen *ConfigGeneratorImpl,
 				Port:     15006,
 				Protocol: protocol.HTTP,
 			},
+			class:    istionetworking.ListenerClassSidecarInbound,
```

Comment:
Good, it's worth a test.

Reply:
I have another PR refactoring all the tests. Will merge that and update the tests

Reply:
done